lib/config.js

```diff
@@ -125,7 +125,7 @@ class Config {
       if (callback) {
         callback.call(this);
       }
-    } else if (configFile.match(/\.js$/)) {
+    } else if (configFile.match(/\.c?js$/)) {
```

Comment:
I haven't looked in detail at the `readJs` method, but is there a reason we can't also support mjs?

Reply:
It'd be a little more involved — you need to use dynamic import instead of `require`. I thought I'd do the minimal change, and someone else can pick up the `.mjs` thing.